src/data/plants/toxic.json

```diff
@@ -92,19 +105,48 @@
       "lat": "strelitzia reginae"
     },
     "slug": "strelicja",
-    "thumbnailID": "866733515",
-    "danger": 4
+    "imageID": [
+      "866733515",
+      "4814038355",
+      "2524917491",
+      "17563691411",
+      "8160264472",
+      "5624276430"
+    ],
+    "danger": 1,
+    "symptoms": "łagodne mdłości, wymioty, ospałość",
+    "note": "głównie owoce i nasiona",
+    "source": [
+      "https://www.aspca.org/pet-care/animal-poison-control/toxic-and-non-toxic-plants/bird-paradise-flower",
+      "https://plants.ces.ncsu.edu/plants/strelitzia-reginae/",
+      "https://pets.thenest.com/birds-paradise-cats-11340.html"
+    ]
   },
   {
     "name": {
       "pl": ["aloes"],
-      "en": ["aloe vera"],
+      "en": ["aloe vera", "Barbados aloe", "the want of heaven"],
       "lat": "aloe vera"
     },
     "queryImage": "aloe vera leaves",
     "slug": "aloes",
-    "thumbnailID": "50658583448",
```

Comment:
the thumbnail image turned out not to be aloe vera 🤷🏻‍♀️
double-double checking

Reply:
<img width="1163" alt="Screenshot 2020-12-26 at 00 32 15" src="https://user-images.githubusercontent.com/9083292/103143207-d34d3280-4711-11eb-964d-1423da66d636.png">